packages/auth/src/accessToken/routes.ts

```diff
@@ -79,10 +81,27 @@ async function revokeToken(
 ): Promise<void> {
   //TODO: verify accessToken with httpsig method
 
-  const revocationError = await deps.accessTokenService.revoke(ctx.params['id'])
-  if (revocationError) {
-    return ctx.throw(404, revocationError.message)
+  await deps.accessTokenService.revoke(ctx.params['id'])
+  ctx.status = 204
+}
+
+async function rotateToken(
+  deps: ServiceDependencies,
+  ctx: AppContext
+): Promise<void> {
+  //TODO: verify accessToken with httpsig method
+  const result = await deps.accessTokenService.rotate(
+    ctx.params['managementId']
+  )
+  if (result.success) {
+    ctx.status = 200
+    ctx.body = {
+      access_token: {
+        access: [accessToBody(result.access)]
+      }
+    }
```

Comment:
Isn't that missing `value`, `manage`, and `expires_in`? I realize that the spec don't say that `manage` and `expires_in` are required but I think they should. I'll include that in my spec updates.

Reply:
PR to update the spec: https://github.com/interledger/open-payments/pull/140